content/en/docs/3.features/6.configuration.md

```diff
@@ -78,18 +78,14 @@ To use these pre-processors, we need to install their webpack loaders:
 ::code-group
 ```bash [Yarn]
 yarn add -D pug pug-plain-loader
-yarn add -D sass sass-loader@10 fibers
+yarn add -D sass sass-loader@10
 ```
 ```bash [NPM]
 npm install --save-dev pug pug-plain-loader
-npm install --save-dev sass sass-loader@10 fibers
+npm install --save-dev sass sass-loader@10
 ```
 ::
 
-::alert{type="info"}
-Synchronous compilation with `sass` (2x speed increase) [is enabled automatically](https://github.com/webpack-contrib/sass-loader) when `fibers` is installed.
-::
```

Comment:
I also removed the note about the benefits of installing node-fiber because I changed it not to install node-fiber